tests/test_processCcd.py

```diff
@@ -164,8 +164,8 @@ def testProcessCcd(self):
                         ("summary_decl", summary.decl, -9.800258687592303),
                         ("summary_zenithDistance", float('%.6f' % (summary.zenithDistance)), 42.361001),
                         ("summary_zeroPoint", summary.zeroPoint, 30.940228147639207),
-                        ("summary_skyBg", summary.skyBg, 191.4352211728692),
-                        ("summary_skyNoise", summary.skyNoise, 12.512330367008024),
+                        ("summary_skyBg", summary.skyBg, 191.37726892903447),
+                        ("summary_skyNoise", summary.skyNoise, 12.512272992606531),
```

Comment:
These are the only two that changed after the (significant!) move of the task from `characterizeImage` to `calibrate`?!  Is the test image basic enough that no other values in the summary stats got modified by `calibrateTask`?  Also, can you now add checks for the newly added astrometry metrics as part of the test?

Reply:
These are the only two that change.  And unfortunately, I can't check the astrometry metrics because the astrometry task isn't run as part of `test_processCcd`.  (Nor should it, since that would add quite a bit of testing overhead.)